controllers/storagecluster/generate.go

```diff
@@ -118,6 +133,17 @@ func generateCephReplicatedSpec(initData *ocsv1.StorageCluster, poolType string)
 	return crs
 }
 
+// generateCephReplica1Spec returns the ReplicatedSpec for the cephCluster
+// based on the StorageCluster configuration
+func generateCephReplica1Spec(initData *ocsv1.StorageCluster) cephv1.ReplicatedSpec {
```

Comment:
nit: How about this method should generate the entire blockpool spec instead of just the replicated spec? Or if simpler, just set these properties in the caller and no need for this method
```suggestion
func generateReplica1BlockPoolSpec() cephv1.ReplicatedSpec {
```